### DISCUSSION TITLE
Mimir Metamonitoring Remote Auth for AzureAD or Managed Identity?

### DISCUSSION THREAD

**Discussion by mackling101** [grafana/mimir#10150] [Help and support]
I am attempting to remote-write the metamonitoring to an Azure Managed Prometheus endpoint, is anyone aware of the appropriate yaml formatting (if it exists) for the metamonitoring authentication?

``` 
remote:
  auth:
    azuread: ?       
```

Is there a way we can authenticate utilizing azuread/app registration or managed identity in some way?

**Comment by mackling101** [comment]
Is anyone aware of implementing mimir within AKS and utilizing the metamonitoring metrics with the Azure Prometheus?

 I've had issues with the prometheus node_exporter otherwise, and am looking to utilize the Azure Prometheus metrics. I believe the issue is related to the authentication, as the azure prometheus does not utilize "basic auth", and needs to utilize azuread or managed identity.

**Reply by colega** [reply]
I'm not familiar with Azure Prometheus, but I was wondering if this is something you could solve with some kind of a reverse-proxy that would do the proper auth with Azure in the middle?

**Reply by mackling101** [reply]
That is an interesting thought. Let me explore that and get back to you.

**Comment by mackling101** [comment]
cries in Azure

**Comment by mackling101** [comment]
So I ended up not doing anything with this and just configured scrape settings. It was simpler to deploy custom scrape settings within Azure Managed Prometheus than it was to try and figure out Auth for remote write.
